### ISSUE TITLE
Usage1：CUDA_VISIBLE_DEVICES=0 python3 main.py --c  ./Config/config.cfg  --t_model bilstm --train -p

### ISSUE THREAD

**Issue by feiyangdezhu** [bamtercelboo/pytorch_text_classification#2]
Usage2：CUDA_VISIBLE_DEVICES=0 python3 main.py --c  ./Config/config.cfg  --t_model cnn --train -p
Both CNN and bilstm models were run on the same data set, but the best results I got were both The Current Best Dev Accuracy: 82.110092, Locate on 12 Epoch.
The Current Best Test Accuracy: accuracy = 84.404174%

**Comment by dalinvip** [comment]
CUDA_VISIBLE_DEVICES=0 python3 main.py --c ./Config/config.cfg  --train -p
modify config.cfg for choose cnn or bilstm, see Config readme

**Comment by feiyangdezhu** [comment]
Both models ran separately, but the results were the same

**Comment by feiyangdezhu** [comment]
All the same

**Comment by feiyangdezhu** [comment]
The code I ran and the data set I used was the one you put on github. I didn't make any changes.

**Comment by dalinvip** [comment]
modify config.cfg to choose model

model_cnn = True
model_bilstm = False

**Comment by feiyangdezhu** [comment]
Thank you, I see

**Comment by feiyangdezhu** [comment]
My best dev set accuracy is 2% lower than you

**Comment by dalinvip** [comment]
test set accuracy is same as me ?
